interface/forms/observation/save.php

```diff
@@ -14,125 +14,33 @@
  */
 
 require_once(__DIR__ . "/../../globals.php");
+/**
+ * @global string $srcdir
+ */
 require_once("$srcdir/api.inc.php");
 require_once("$srcdir/forms.inc.php");
 
-use OpenEMR\Common\Csrf\CsrfUtils;
-
-if (!CsrfUtils::verifyCsrfToken($_POST["csrf_token_form"])) {
-    CsrfUtils::csrfNotVerified();
-}
-
-if (!$encounter) { // comes from globals.php
-    die(xlt("Internal error: we do not seem to be in an encounter!"));
+use Symfony\Component\HttpFoundation\Request;
+use OpenEMR\Controllers\Interface\Forms\Observation\ObservationController;
+use OpenEMR\Services\ObservationService;
+use OpenEMR\Common\Logging\SystemLogger;
+
+$logger = new SystemLogger();
+
+try {
+    // Create controller and handle request
+    $request = Request::createFromGlobals();
+    $service = new ObservationService();
+    $controller = new ObservationController($service);
+    $response = $controller->save($request);
+    $response->send();
+    exit;
```

Comment:
`exit;`s are unconditional here. If you can't get rid of them, you could unify them below the block.

Reply:
Should be safe to just remove them.